Resources/Prototypes/Entities/Objects/Weapons/Guns/Basic/base_pka.yml

```diff
@@ -25,7 +25,7 @@
           False: { visible: True }
   - type: RechargeBasicEntityAmmo
     minRechargeCooldown: 1.5
-    maxRechargeCooldown: 3
+    maxRechargeCooldown: 1.51
```

Comment:
can you not just set both to 1.5, pretty sure that would work fine?

Reply:
was afraid it'd cause issues but it seems ok